Jamroot.jam

```diff
@@ -0,0 +1,20 @@
+path-constant LIBTORRENT_ROOT : . ;
```

Comment:
is there a good reason to put this rule in Jamroot.jam instead of Jamfile (all the other similar rules are in Jamfile)